llarp/win32/windivert.cpp

```diff
@@ -106,18 +192,29 @@ namespace llarp::win32
           throw win32::error{
               err, fmt::format("failed to receive packet from windivert (code={})", err)};
         }
-        log::trace(logcat, "got packet of size {}B", sz);
         pkt.resize(sz);
+
+        log::trace(logcat, "got packet of size {}B", sz);
+        log_windivert_addr(addr);
+
         return Packet{std::move(pkt), std::move(addr)};
       }
 
       void
-      send_packet(const Packet& w_pkt) const
+      send_packet(Packet&& w_pkt) const
```

Comment:
this does not need to be an R value reference. or any other reference for that matter.